keyboards/40percentclub/half_n_half/rules.mk

```diff
@@ -62,15 +62,15 @@ BOOTLOADER = caterina
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration(+1000)
-MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration(+1000)
```

Comment:
Ideally, these shouldn't be changed from the default. 

You can create a rules.mk fine on your keymap folder, and this will override the values there.